apps/admin-server/src/components/ui/sortTable.tsx

```diff
@@ -9,6 +9,7 @@ const sortFunctions = {
     'voted-yes': (a: any, b: any) => b.resource?.yes || 0 - a.resource?.yes || 0,
     'voted-no': (a: any, b: any) => b.resource?.no || 0 - a.resource?.no || 0,
     'name': (a: any, b: any) => a.name.toLowerCase().localeCompare(b.name.toLowerCase()),
+    'url': (a: any, b: any) => b.name.toLowerCase().localeCompare(a.name.toLowerCase()),
```

Comment:
Dit lijkt de omgekeerde versie van `name` hierboven. Klopt dit wel?

Reply:
Was wel bewust, maar nu ik erover nadenk is het niet nodig. Was omdat de volgorde verkeerd om leek onClick. Maar die andere velden waren leeg.